platforms-serverless/serverless-framework-lambda/prisma/schema.prisma

```diff
@@ -1,7 +1,7 @@
 generator client {
+  output = "client"
   provider      = "prisma-client-js"
-  // second binaryTarget `rhel` is required for deploying to AWS Lambda
-  binaryTargets = ["native", "rhel-openssl-1.0.x"]
```

Comment:
Removing this makes it not work on local Windows/Mac/Ubuntu any more etc.

Reply:
Yes, I think that needs rework (I was lazy), somehow the excluded engine is still included. I take a look.

Reply:
Probably because of the new custom output.